packages/site-kit/components/Nav.svelte

```diff
@@ -20,10 +20,15 @@
 		open = false;
 	});
 
+	let window_inner_width;
+	function is_mobile() {
+		if (window_inner_width < 768) return true;
```

Comment:
```js
if () {
    return true;
}
return false;
```

is nonsense,... you can just simply write:

`return (window_inner_width < 768)`